src/test/java/core/basesyntax/MyHashMapTest.java

```diff
@@ -1,57 +1,86 @@
+        Assert.assertNull(myHashMap.getValue(firstCar));
     }
 
     @Test
-    public void getTheOverriddenValueByKey() {
-
+    public void putAndGetOk() {
+        MyMap<Car, Integer> myHashMap = new MyHashMap();
+        myHashMap.put(firstCar, 3);
+        myHashMap.put(secondCar, 5);
+        myHashMap.put(thirdCar, 1);
+
+        Assert.assertEquals(3, myHashMap.getSize());
+        Assert.assertEquals(Integer.valueOf(3), myHashMap.getValue(firstCar));
+        Assert.assertEquals(Integer.valueOf(5), myHashMap.getValue(secondCar));
+        Assert.assertEquals(Integer.valueOf(1), myHashMap.getValue(thirdCar));
     }
 
     @Test
-    public void putOk() {
-
+    public void putAndGetByNullKey() {
+        MyMap<Car, Integer> myHashMap = new MyHashMap();
+        myHashMap.put(null, 3);
+        Assert.assertEquals(Integer.valueOf(3), myHashMap.getValue(null));
+        Assert.assertEquals(1, myHashMap.getSize());
+        myHashMap.put(null, 5);
+        Assert.assertEquals(Integer.valueOf(5), myHashMap.getValue(null));
+        Assert.assertEquals(1, myHashMap.getSize());
     }
 
     @Test
-    public void putByNullKey() {
-
+    public void putAndGetWithCollision() {
+        MyMap<Plane, Integer> myHashMap = new MyHashMap();
+        myHashMap.put(firstPlane, 3);
+        myHashMap.put(secondPlane, 5);
+        myHashMap.put(thirdPlane, 1);
+
+        Assert.assertEquals(3, myHashMap.getSize());
+        Assert.assertEquals(Integer.valueOf(3), myHashMap.getValue(firstPlane));
+        Assert.assertEquals(Integer.valueOf(5), myHashMap.getValue(secondPlane));
+        Assert.assertEquals(Integer.valueOf(1), myHashMap.getValue(thirdPlane));
     }
 
     @Test
-    public void putWithCollision() {
-        //TODO: use PLane
+    public void putAndGetTheOverriddenValueByKey() {
+        MyMap<Car, Integer> myHashMap = new MyHashMap();
+        myHashMap.put(firstCar, 3);
+        Assert.assertEquals(1, myHashMap.getSize());
+        Assert.assertEquals(Integer.valueOf(3), myHashMap.getValue(firstCar));
+
+        myHashMap.put(firstCar, 5);
+        Assert.assertEquals(1, myHashMap.getSize());
+        Assert.assertEquals(Integer.valueOf(5), myHashMap.getValue(firstCar));
     }
 
-
     @Test
-    public void putTheOverriddenValueByKey() {
-
+    public void checkTheHashMapIncrease() {
+        MyMap<Car, Integer> myHashMap = new MyHashMap();
+        for (int i = 0; i < 1000; i++) {
+            Car car = new Car("model_" + i, "color_" + i);
+            myHashMap.put(car, i);
+        }
+        Assert.assertEquals(1000, myHashMap.getSize());
+        Assert.assertSame(999, myHashMap.getValue(new Car("model_999", "color_999")));
     }
 
     @Test
-    public void checkTheHashMapIncrease() {
-        //TODO: add 1000 elements
+    public void getSizeOfEmptyHashMap() {
+        MyMap<Car, Integer> myHashMap = new MyHashMap();
+        Assert.assertEquals(0, myHashMap.getSize());
```

Comment:
@Sofasmile add more test cases to check the size of HashMap. Especially with collisions